test/lib/code_corps/github/adapters/user_test.exs

```diff
@@ -20,15 +31,15 @@ defmodule CodeCorps.GitHub.Adapters.UserTest do
     end
 
     test "maps Bot type" do
-      assert User.to_user(%{"type" => "Bot"}) == %{type: "bot"}
+      assert User.to_user(%{"type" => "Bot"}) == expected_payload("bot")
     end
 
     test "maps User type" do
-      assert User.to_user(%{"type" => "User"}) == %{type: "user"}
+      assert User.to_user(%{"type" => "User"}) == expected_payload("user")
     end
 
     test "does not map Organization type" do
-      assert User.to_user(%{"type" => "Organization"}) == %{type: "Organization"}
+      assert User.to_user(%{"type" => "Organization"}) == expected_payload("Organization")
```

Comment:
I'm seeing some inconsistencies here which I'm unsure about.

Any idea why the other two are tested with lowercase, while this one is uppercase?

Reply:
I am not sure what's reason, but `Github.Adapters.Users` enforces lowercase `user` and `bot` type, but not `Organization`. @joshsmith may know more.

Reply:
Hmm I'm not sure there's any good reason for that. We probably should enforce it, too.

Reply:
@iref Would you be ok with trying to add that enforcement as part of this PR? We may have a potential bug or at least inconsistency and I'm hesitant to merge until we know more.

Reply:
Type normalization isn't related to original issue, so new PR might be better. Just to keep changeset as small as possible. Although I don't feel super strong about it. :)

Do we want to normalize any type value in future? This change wouldn't need additional changes if Github adds new user type, but I don't know if it's desired behaviour or we rather normalize types per case.

Reply:
@joshsmith What do you think? It's definitely out of scope for this PR, so I'd be fine with a new issue, but it's something we probably ought to consolidate.

It might involve migrating data to. Do we ever store "users" of type "Organization"?

Reply:
I think I’ve seen this come up. Honestly it’s something that should also be normalized at the database level too in these cases by lower casing the values in the database.

In which case I think this should be dealt with as a separate issue.